packages/contracts/contracts/plugin/examples/count/CounterV1PluginSetup.sol

```diff
@@ -20,7 +20,7 @@ contract CounterV1PluginSetup is PluginSetup {
     MultiplyHelper public multiplyHelperBase;
     CounterV1 public counterBase;
 
-    address private constant NO_ORACLE = address(0);
+    address private noOracle;
```

Comment:
does it make sense to save gas during deployment but use more during the lifetime of the contract?
using constant will not reserve a storage slot and therefore doesn't use a storage read during execution.

Reply:
thanks